src/component/registry/types.ts

```diff
@@ -191,7 +199,11 @@ export type Stroke = StyleWithOpacity & {
 export type Font = StyleWithOpacity & {
   /**
    * Possible values are all HTML color names or HEX codes, as well as special keywords such as:
-   * - `default` to use the color defined in the BPMN element default style. **WARN**: this doesn't use the color set in the BPMN source when the "BPMN in Color" support is enabled.
+   * - `default` to use the color defined in the BPMN element default style.
+   *
+   * Notes about the `default` special keyword:
```

Comment:
**suggestion:** 
To keep consistency with the APIs of `BpmnElementsRegistry`:
```suggestion
   * **Notes about the `default` special keyword**:
```

Reply:
done in 99f56ddb